baremetal.go

```diff
@@ -0,0 +1,114 @@
+// Package vagrantssh provides host connectivity in go for system/integration
+// testing in a multi host environment. It supports two testbed environments viz.
+// baremetal and vagrant
+/*
```

Comment:
put the copyright notice after the package declaration. it'll show up better in the docs too.